modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -3,46 +3,48 @@
-
-Cluster balancing protects you from unbalanced systems that saturate resources on one or more brokers. This can affect throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss.
-
-The following table summarizes the various balancing options in Redpanda:
-
-
-.Cluster balancing options
-[cols="4,2,11"]
+[.no-clip]
 |===
-|Balancer |xref:get-started:licenses.adoc[License] |Description
+| Balancer |Description
 
-|Partition leadership balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+| <<Partition leadership balancing>>
+| Topic-aware balancing that transfers the leadership of a broker's partitions to other replicas. This helps avoid topic leadership hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+Redpanda automatically provides leadership balancing when brokers are added or decommissioned.
+
+| <<Partition replica balancing>>
+| Topic-aware balancing that moves partition replicas. This helps avoid topic replica hotspots on one or a few specific brokers in your cluster. 
+ +
+ +
+Redpanda prioritizes balancing a topic's partition count on each broker while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers.
+ +
+ +
+Redpanda automatically provides partition replica balancing when brokers are added or decommissioned.
 
-|Partition replica balancing
-|Community Edition, Enterprise Edition
-|Topic-aware balancer designed to distribute partition replicas when adding nodes to the cluster. This helps avoid topic replica hotspots on one or a few specific nodes in your cluster.
+| xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing]
 
-|Continuous Data Balancing
-|Enterprise Edition
-|Balancer designed to monitor free disk space and proactively move data between nodes to avoid exhausting available disk space on any given node. This balancer does not keep the relative fullness of each node within a defined range, it just prevents hitting the fullness threshold of each individual node.
+| With an Enterprise license, you can additionally enable Continuous Data Balancing to continuously monitor broker and rack availability and disk usage. This enables self-healing clusters that dynamically balance partitions. It also ensures adherence to rack-aware replica placement policy and self-heals after rack (or availability zone) failure or replacement. This balancer does not keep the relative fullness of each broker within a defined range, it just prevents hitting the fullness threshold of each individual broker. 
```

Comment:
@micheleRP  I see why we have this structure to differentiate community/enterprise, but we're missing that the _continuous_ data balancing (which btw is partition replica balancing) is ALSO topic-aware. (as is the continuous  leadership balancing).

We may want to rename the previous sections to 'topic-aware data balancing' just to simplify things.    

And we certainly need to call out that the continuous mode also uses topic-aware balancing to avoid topic hotspots when moving data off of brokers whose disks are more full

Reply:
Thanks, please review the [updated section](https://deploy-preview-416--redpanda-docs-preview.netlify.app/24.1/manage/cluster-maintenance/cluster-balancing/).